src/sql/platform/theme/common/colorRegistry.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { registerColor } from 'vs/platform/theme/common/colorRegistry';
+import { Color, RGBA } from 'vs/base/common/color';
+import * as nls from 'vs/nls';
+
+// -- Welcome Page Colors
+export const tileBoxShadowColor = new Color(new RGBA(0, 1, 4, 0.13));
+export const textShadow = new Color(new RGBA(0, 0, 0, 0.25));
+export const dropdownBoxShadow = new Color(new RGBA(0, 0, 0, 0.25));
+export const extensionPackGradientOne = new Color(new RGBA(50, 49, 48, 0.55));
+export const extensionPackGradientTwo = new Color(new RGBA(50, 49, 48, 0));
+export const gradientOneColorOne = new Color(new RGBA(0, 0, 0, .2));
+export const gradientTwoColorOne = new Color(new RGBA(156, 48, 48, 0));
+export const gradientTwoColorTwo = new Color(new RGBA(255, 255, 255, 0.1));
+
+// -- Tiles
+export const tileBorder = registerColor('tileBorder', { light: '#fff', dark: '#8A8886', hc: '#2B56F2' }, nls.localize('tileBorder', "The border color of tiles"));
+export const tileBoxShadow = registerColor('tileBoxShadow', { light: tileBoxShadowColor, dark: tileBoxShadowColor, hc: tileBoxShadowColor }, nls.localize('tileBoxShadow', "The tile box shadow color"));
+
+// -- Buttons
+export const buttonSecondaryBorder = registerColor('button.secondaryBorder', { light: '#8A8886', dark: '#FFF', hc: '#264BD3' }, nls.localize('button.secondaryBorder', "The border color for secondary button"));
```

Comment:
If you are adding new colors you should make a point to add good colors for all our built in themes.